addons/binding/org.openhab.binding.russoundserial/README.md

```diff
@@ -0,0 +1,98 @@
+#Russound Serial Binding
+
+This binding allows you to control a [Russound MCA/CAA](https://www.russound.com/products/audio-systems/multi-room-controllers/caa66-system-and-kits/caa66-controller-amplifier) system over a
+serial connection. There is an existing [Russound plugin](https://github.com/openhab/openhab2-addons/tree/master/addons/binding/org.openhab.binding.russound), but that works over a network connection on a different protocol (RIO).
+
+This plugin uses a [kotlin library]([https://github.com/holmes/russound) to communicate via the serial protocol.
+
+## Supported Things
+
+There are 2 supported Thing(s):
+1. Bridge: This is your connection to the controller. You'll specify the serial port once per bridge.
+1. Zones: You'll have a 1:1 mapping of a Zone Thing to a physical zone on your Russound device.
+
+## Discovery and Thing Configuration
+
+There is no auto-discovery. 
+
+#### Bridge
+You must specify the serialPort the Bridge is connected to when setting up the Bridge Thing.
+
+#### Zones
+You must specify the zoneNumber (0-based) this Thing is targeting. This maps directly to the number on the back of the device. 
+
+## Channels
+
+Name       | Type | Description
+---|---|---
+zonePower | Switch | Whether the zone is ON or OFF
+zoneVolume | Dimmer | Volume level of zone 
+zoneSource | Number | The source that's currently playing (0-based)
+zoneBass | Number | Bass Setting (-10 to 10)
+zoneTreble | Number | Treble Setting (-10 to 10)
+zoneBalance | Number | Balance Setting (-10 full left to 10 full right)
+zoneLoudness | Switch | Whether the loudness setting is on or off
+zoneTurnOnVolume | Dimmer | This is a write only setting! Determines the volume when the zone is turned on
+
+## Full Example
```

Comment:
You can have a look here:
http://docs.openhab.org/configuration/things.html#defining-things-using-files

or the README files of other bindings also include a full example as well.